lms/lmsweb/translations/he/LC_MESSAGES/messages.po

```diff
@@ -29,7 +29,7 @@ msgstr "הפתרון שלך לתרגיל %(subject)s נבדק."
 
 #: lms/lmstests/public/linters/services.py:95
```

Comment:
Yay, thanks! But please notice that you need to change these strings in the project's `.py` and `.html` files as well